pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -71,3 +75,6 @@ cdef int64_t pydate_to_dt64(date val, pandas_datetimestruct *dts)
 cdef npy_datetime get_datetime64_value(object obj) nogil
 cdef npy_timedelta get_timedelta64_value(object obj) nogil
 cdef PANDAS_DATETIMEUNIT get_datetime64_unit(object obj) nogil
+
+cdef int _string_to_dts(object val, pandas_datetimestruct* dts,
```

Comment:
can deprivatize at some point

Reply:
Yah I’m currently running asv on a proposed change to string_to_dts that would make to_datetime and Timestamp behavior more similar (or at least make the dissimilarity more obvious)